src/containers/PageBuilder/Primitives/SearchCTA/SearchCTA.module.css

```
@@ -0,0 +1,123 @@
+@import '../../../../styles/customMediaQueries.css';
+
+.filterField {
+  height: 100%;
+  align-items: center;
+  display: grid;
+  min-height: 40px;
+  /* exp */
+  padding-bottom: 8px;
+  padding-top: 8px;
+  @media (--viewportLarge) {
+    padding-bottom: 0;
+  }
+}
+
+.align {
+  justify-self: start;
+  @media (--viewportMedium) {
+    justify-self: center;
+  }
+}
+
+.searchBarContainer {
+  composes: align;
+  background-color: var(--colorWhite);
+  border-color: var(--colorGrey800);
+  border-width: 1px;
+  color: var(--colorGrey500);
+  border-radius: 8px;
+  box-shadow: var(--boxShadowPopup);
+  width: 100%;
```

Something like this perhaps:
```suggestion
  width: calc(100vw - 72px);
```
or `width: 80vw,` but this is not as accurate with the margins.

If the value is "100%", the item inside a grid is taking the content width instead of grid's width.